OpenAPI/kiota/install.md

```diff
@@ -128,12 +128,15 @@ dotnet tool install --global Microsoft.OpenApi.Kiota
 
 ## Install the Visual Studio Code extension
 
-1. Open the [Marketplace page of the extension](https://aka.ms/kiota/extension)
-1. Select on the **Install** button.
+1. Open VS Code and go to the [Marketplace page of the extension](https://aka.ms/kiota/extension)
+2. Click **Install**.
+3. Once Kiota is installed, enable the integration with Teams Toolkit (TTK) to leverage Kiota for creating AI plugins.
```

Comment:
We still have a large customer base who only generate API clients, I'd suggest we move this last point to a separate section below. And add another bullet point to tell them to install TTK extension in case they don't have it.

Reply:
Hi @baywet, this PR isn't ready for review. I am adding the info and then giving it a more coherent flow. Thanks for the feedback, though. I'll give you a heads-up once I'm done. :)

Reply:
Ah thanks for letting me know. Don't hesitate to switch the PR to draft, and back to ready when... Ready